### PR TITLE
Submit text memory to db

### PR DESCRIPTION
**Pull request by joeylouise**
**Merge #61 first please**

This PR:

Reorders db_build.sql so that build builds correctly
Adds routes to create memory in db for text memory

Relates #57 #59

### CODE REVIEW

**Comment by des-des** [comment]
Merge and re-assign -- this has commits from other pulls on it!!

**Comment by joeylouise** [comment]
@des-des fixed now!

File: src/routes/addTextMemory.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+require('env2')('./config.env');
+const { createMemory } = require('./../database/db_create');
+
+const app = express.Router();
+
+app.use(bodyParser());
```
Comment (des-des) [inline src/routes/addTextMemory.js:8]: No, this should be in the root file

File: src/routes/addTextMemory.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+module.exports = [
+  app.post('/memory-input-text', (req, result) => {
+    createMemory(req.body, 'text_only', (err, res) => {
+      // 1 hard coded in at the moment.
+      // We will later add the userId here to get the relevant memories.
+      if (err) {
+        return err;
+      }
+      setTimeout(() => {
+        result.redirect('/');
```
Comment (des-des) [inline src/routes/addTextMemory.js:19]: assuming this is temporary?

**Review by des-des — CHANGES_REQUESTED** [review]
Coolio,

You do not need to do it in this pull, but check out
https://github.com/pbywater/echo/issues/70

File: src/routes/addTextMemory.js (hunk 1 above)
Comment (pbywater) [reply src/routes/addTextMemory.js:8]: @joeylouise - router bits have been fixed in deleting-node branch

**Review by des-des — APPROVED** [review]

**Pull request merged**